lessonbuilder/tool/src/webapp/js/checklist.js

```diff
@@ -182,7 +182,7 @@
             return;
         }
         var obj = $.parseJSON(attributeString);
-        if (obj) {
+        if (obj && obj.checklistItems !== null) {
```

Comment:
Does this make more sense?
`if (typeof obj !== 'undefined')`

Reply:
no, I don't think so because obj _is_ defined.  It's the obj.checklistItems that is null when the failure occurs

![obj](https://user-images.githubusercontent.com/28317132/50661624-c55fdf00-0f47-11e9-95f3-2f689fc6893b.png)